### ISSUE TITLE
Wrong link to source forum post on my node

### ISSUE THREAD

**Issue by MrPetovan** [friendica/friendica#5717]
### Expected behavior

Looking at this URL: https://friendica.mrpetovan.com/display/735a2029155b82041d45e7c190315959

The link to source should be https://forum.friendi.ca/display/735a2029155b82041d45e7c190315959

### Actual behavior

Instead it is https://friendica.mrpetovan.com/redir/6?url=display/735a2029155b82041d45e7c190315959

`6` is the contact id for my self public contact.

### Steps to reproduce the problem

Post to a forum with `!`, although exact circumstances are unknown.

### Friendica version you encountered the problem

2018-08-rc

### Friendica source (git, zip)

git

### PHP version

7.2

### SQL version

MariaDB 10.10

**Comment by annando** [comment]
Yeah, I know. We are not totally acting clean on creating forum posts, so to say ;-)

But I would prefer not to change that part of the code that short before the release.

**Comment by MrPetovan** [comment]
I didn't add this to the current milestone for this reason.
